.travis.yml

```
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "8.11.4"
+  - 12
```

This will test with both `lts` and `current`

```suggestion
  - 10.13
  - 12
```

---

Set `lts` (10) and `current` (12) versions for nodejs